atm.js

```diff
@@ -1,3 +1,121 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+// define savings/checking account
+var $savingsBalance = $("#savings .balance");
+var $checkingBalance = $("#checking .balance");
+var $accountBalance;
+var $otherAccountBalance;
+
+// other variables
+var $accountInput;
+var updatedBalance;
+var otherUpdatedBalance;
+var $balance = $(".balance");
+var $depositBtn = $(".deposit");
+var $withdrawBtn = $(".withdraw");
+
+
+// starting balance with .zero
+$balance.addClass("zero");
+
+// on click of deposit or withdraw, user adds/subtracts balance
+$depositBtn.on("click", depositSomething);
+$withdrawBtn.on("click", withdrawSomething);
+
+
+function depositSomething(e) {
```

Comment:
maybe leave off Something for variable name, it's a small thing, but something isn't really descriptive and deposit is enough semantics to clarify what the functions trying to do.